vm/src/objtype.rs

```diff
@@ -49,6 +49,11 @@ pub fn isinstance(obj: PyObjectRef, cls: PyObjectRef) -> bool {
     mro.into_iter().any(|c| c.is(&cls))
 }
 
+pub fn issubclass(typ: PyObjectRef, cls: PyObjectRef) -> bool {
+    let mro = _mro(typ).unwrap();
+    mro.into_iter().any(|c| c.is(&cls))
```

Comment:
Do you also need to check if typ itself is cls?

Reply:
That's a very good question. I do not know. I thought of leaving this and facing the issue later :blush:

Reply:
Yes, but issubclass means that it is either of its own type, or of one of its inherited types.